### PR TITLE
Fix Build Error for iMXRT1020 configuration

### PR DESCRIPTION
**Pull request by HarrisonChoi750405**

### CODE REVIEW

**Comment by seinfra** [comment]
Target : [705caa9a71f3a4248d23a6dc26148547563352a1] - Code Rule Check (C++) OK.

**Comment by seinfra** [comment]
Target : [705caa9a71f3a4248d23a6dc26148547563352a1] - Code Rule Check Result:
os/arch/arm/src/imxrt/imxrt_pwm.c:936: ERROR: [SPC_M_OPR] space required after that ',' (ctx:VxV)
os/arch/arm/src/imxrt/imxrt_pwm.c:1186: ERROR: [IDT_M_SCH] switch and case should be at the same indent

**Comment by seinfra** [comment]
Target : [8f3cd61047cda982f55d19d03ff166a31ede7267] - Code Rule Check (C++) OK.

**Comment by seinfra** [comment]
Target : [8f3cd61047cda982f55d19d03ff166a31ede7267] - Code Rule Check OK.

File: os/arch/arm/src/imxrt/imxrt_norflash.c [hunk 1]
```diff
@@ -935,7 +935,7 @@ ssize_t up_progmem_write(size_t addr, const void *buf, size_t count)
 {
 	printf("up_progmem_write\n");
 
-	return flexspi_nor_flash_write(IMXRT_FLEXSPI, addr, (const uint32_t *)buf, count);
+	return 0;
```
Comment (sunghan-chang) [inline os/arch/arm/src/imxrt/imxrt_norflash.c]: Is this temporary fix? or Does not 1020-evk support up_progmem_write?

File: os/arch/arm/src/imxrt/imxrt_mtd_flash.c [hunk 2]
```diff
@@ -339,11 +339,15 @@ static inline void imxrt_bytewrite(struct imxrt_dev_s *priv, FAR const uint8_t *
 	status_t status;
 	fvdbg("offset: %08lx  count:%d\n", (long)offset, count);
 
+	#if defined(CONFIG_ARCH_CHIP_FAMILY_IMXRT102x)
+	#warning "Need to check Configuration!!"
```
Comment (sunghan-chang) [inline os/arch/arm/src/imxrt/imxrt_mtd_flash.c]: same as previous

**Pull request merged**